pontoon/gpt/openai_connection.py

```diff
@@ -0,0 +1,56 @@
+from openai import OpenAI
+
+
+class OpenAIService:
+    def __init__(self):
+        self.client = OpenAI()
+
+    def get_translation(self, english_text, slovenian_text, formality):
+        # Define system messages for different translation tasks
+        system_messages = {
+            "informal": """You will be provided with a passage in English, along with its machine-generated translation in Slovenian. Your objective is to revise the Slovenian translation to ensure it utilizes simpler language. Please adhere to the following guidelines to achieve this:
+                    - Simplify Vocabulary: Use common, everyday words that are easily understood by a broad audience, avoiding technical jargon, idiomatic expressions, and complex terms.
+                    - Shorten and Simplify Sentences: Break down long, complex sentences into shorter, more manageable ones. Aim for clarity and conciseness in each sentence.
+                    - Use Basic Grammar Structures: Avoid complex grammatical constructions. Stick to simple tenses and straightforward sentence structures.
+                    - Clarity is Key: Ensure the translation conveys the original message in the clearest possible manner, without ambiguity or unnecessary complexity.
+                    - Engage a Wide Audience: The simplified translation should be accessible and easily understandable to people with varying levels of Slovenian proficiency, including young readers and non-native speakers.
+                    - Consistent Simplicity: Maintain a consistent level of simplicity throughout the translation. The aim is to make the text as accessible as possible without sacrificing accuracy or meaning.
+                    The goal is to produce a translation that accurately reflects the original English text, but in a way that is more approachable and easier to understand for all Slovenian speakers, regardless of their language proficiency.""",
+            "formal": """You will be presented with text in English, accompanied by its machine-generated translation in Slovenian. Your task is to refine the Slovenian translation to ensure it adheres to a higher level of formality. In doing so, consider the following guidelines:
+                    - Upgrade Language Use: Elevate the language by selecting more sophisticated vocabulary and phrases that are appropriate for formal contexts.
```

Comment:
I'd be tempted to drop this. One thing is to ask for formal structure ("Sie" vs "Du" in German), but this is going one step further, asking for super-formal vocabulary.

Reply:
I agree. Based on the feedback received from the prompt evaluation I think this guideline may have caused the output to be extremely formal.